python/ray/_raylet.pyx

```diff
@@ -772,9 +774,15 @@ cdef void execute_task(
             function = execution_info.function
 
             if core_worker.current_actor_is_asyncio():
+                def predicate(obj):
+                    return (
+                        inspect.iscoroutinefunction(obj)
```

Comment:
I've seen this predicate a lot. Maybe consider to convert it to a util function?